libpod/container_internal.go

```diff
@@ -1546,6 +1546,14 @@ func (c *Container) pause() error {
 		}
 	}
 
+	if c.state.HCUnitName != "" {
+		if err := c.removeTransientFiles(context.Background(),
+			c.config.StartupHealthCheckConfig != nil && !c.state.StartupHCPassed,
+			c.state.HCUnitName); err != nil {
+			logrus.Errorf("failed to remove HealthCheck timer: %v", err)
```

Comment:
why is this only logged? Would it not be better to return this? Or at least use some form of multi error handling.
logrus lines will not be seen for remote users